kernel/sysfile.c

```diff
@@ -16,6 +18,8 @@
 #include "file.h"
 #include "fcntl.h"
 
+atomic_uint readcount;
```

Comment:
```suggestion
_Atomic uint64 readcount;
```

피드백: `_Atomic` 키워드를 쓰는 것이 좋다. `stdatomic.h` 파일은 libc 이므로.
ref) https://en.cppreference.com/w/c/language/atomic.html

Reply:
https://github.com/EcmaXp/xv6-riscv/pull/1/commits/3e30bffc706af2532b89f92cdf2b0776342bf9a9